### PR TITLE
capitalised the export product button

### LINKED ISSUE #3881: Export products button should be capitalised
image

Please create separate issue for this

_Originally posted by @poulch in https://github.com/saleor/saleor-dashboard/issues/3712#issuecomment-1621763451_

### PR DESCRIPTION
**Pull request by arvindpndit**
closes #3881 

### Screenshots

Before: 
Screenshot from 2023-09-25 21-21-11

After:
Screenshot from 2023-09-25 21-21-18
->

### Pull Request Checklist

1. [x] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `data-test-id` are added for new elements
4. [ ] The changes are tested in Chrome/Firefox/Safari browsers and in light/dark mode
5. [ ] Your code works with the latest stable version of the core
6. [ ] I added changesets and [read good practices](../.changeset/README.md)

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] app
3. [ ] attribute
4. [ ] category
5. [ ] collection
6. [ ] customer
7. [ ] giftCard
8. [ ] homePage
9. [ ] login
10. [ ] menuNavigation
11. [ ] navigation
12. [ ] orders
13. [ ] pages
14. [ ] payments
15. [ ] permissions
16. [ ] plugins
17. [ ] productType
18. [ ] products
19. [ ] sales
20. [ ] shipping
21. [ ] translations
22. [ ] variants
23. [ ] vouchers

CONTAINERS=2

### CODE REVIEW

File: src/products/components/ProductExportDialog/messages.ts [hunk 1]
```diff
@@ -8,8 +8,8 @@ export const productExportDialogMessages = defineMessages({
     description: "export products to csv file, dialog header",
   },
   confirmButtonLabel: {
-    id: "oOFrUd",
-    defaultMessage: "export products",
+    id: "7FL+WZ",
+    defaultMessage: "Export Products",
```
Comment (poulch) [inline src/products/components/ProductExportDialog/messages.ts:12]:
```suggestion
    defaultMessage: "Export products",
```

**Review by poulch — COMMENTED** [review]
Thank you for your contribution. Please fix this one thing.
Additionally please run `npm run change:add` select minor and provide description

File: src/products/components/ProductExportDialog/messages.ts (hunk 1 above)
Comment (arvindpndit) [reply src/products/components/ProductExportDialog/messages.ts]: I have addressed the requested changes. I also ran `npm run change:add` selected minor and provided the required description.

**Review by poulch — APPROVED** [review]

File: src/products/components/ProductExportDialog/messages.ts (hunk 1 above)
Comment (poulch) [reply src/products/components/ProductExportDialog/messages.ts]: Could you run `npm run extract-messages` once again.

File: .changeset/serious-starfishes-develop.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
```
Comment (andrzejewsky) [inline .changeset/serious-starfishes-develop.md:2]: You are fixing small typo in this PR so it's a "patch" - could you change it?

**Review by andrzejewsky — CHANGES_REQUESTED** [review]

File: src/products/components/ProductExportDialog/messages.ts (hunk 1 above)
Comment (arvindpndit) [reply src/products/components/ProductExportDialog/messages.ts]: I ran `npm run extract-messages`, What should i do next?

File: .changeset/serious-starfishes-develop.md (hunk 2 above)
Comment (arvindpndit) [reply .changeset/serious-starfishes-develop.md:2]: Okay

File: src/products/components/ProductExportDialog/messages.ts (hunk 1 above)
Comment (poulch) [reply src/products/components/ProductExportDialog/messages.ts]: Please run once again and if there will be any changes in git, please commit and push

Comment (arvindpndit) [reply src/products/components/ProductExportDialog/messages.ts]: Okay

**Review by poulch — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**